metadata.yaml

```diff
@@ -1,4 +1,5 @@
 name: docker-registry
+display-name: Docker Registry Charm
```

Comment:
Drop the `Charm` suffix.  The `display-name` only comes into play in the charm store, at which point users already know it's a charm.